tests/test_integration.py

```
@@ -149,32 +159,33 @@ def test_mlflow_logging_verification(sample_data, tmp_path, temp_mlflow_home):
-        
+
+    with (
+        patch("creditrisk.models.predict.MODELS_DIR", model_dir),
+        patch("catboost.CatBoostClassifier") as mock_model,
+    ):
+
         # Mock model predictions
-        mock_model.return_value.predict_proba.return_value = \
-            np.random.random((len(test_data), 2))
-        
+        mock_model.return_value.predict_proba.return_value = np.random.random((len(test_data), 2))
```

Ruff (0.8.2)

Undefined name `np`

(F821)